msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,626 @@
+
+### Create an Outlook Add-in project
+
+1. Open a second instance of Visual Studio Code.
+1. With Teams Toolkit open in Visual Studio Code, select **Create a new app**.
+1. In the **Select an option** drop down menu, select **Outlook add-in**, and then select **Taskpane**.
+1. Select the folder where you want to create the add-in.
+1. Give a name (with no spaces) to the project when prompted and Teams Toolkit creates the project with basic files and scaffolding *and open it in a separate Visual Studio Code window*. You will use this project as a source for files and markup that you add to the Teams project.
+1. Although you won't be developing this project, verify that it can be sideloaded from Visual Studio Code before you continue. Use the following steps:
+
+    1. *First, make sure Outlook desktop is closed.*
+    1. In Visual Studio Code, open the Teams Toolkit.
+    1. In the **ACCOUNTS** section, verify that you're signed into Microsoft 365.
+    1. Select **View** | **Run** in Visual Studio Code. In the **RUN AND DEBUG** drop down menu, select the option, **Outlook Desktop (Edge Chromium)**, and then press F5. The project builds and a Webpack dev-server window opens. This process can take a couple of minutes. Eventually, Outlook desktop opens.
+    1. Open the **Inbox** *of your Microsoft 365 account identity* and open any message. A **Contoso Add-in** tab with two buttons appears on the **Home** ribbon (or the **Message** ribbon, if you open the message in its own window).
+    1. Select the **Show Taskpane** button and a task pane opens. Select the **Perform an action** button and a small notification appears near the top of the message.
+    1. To stop debugging and uninstall the add-in, select **Run** | **Stop Debugging** in Visual Studio Code. If the Webpack dev-server window doesn't close, open the Visual Studio Code **TERMINAL** in the root of the project and run `npm stop`.
+
+### Merge the manifest
+
+The Teams app's manifest is generated at debug-and-sideload time (or build time) from the file manifest.json in the \appPackage folder of the Teams project. *This file is actually a kind of **template** for a manifest. In this article it is referred to as the "template" or "manifest template".* Most of the markup is hardcoded into the template, but there are also some configuration files that contain data that gets injected into the final generated manifest. In this procedure, you do the following tasks:
+
+- Copy markup from the add-in's manifest to the Teams app's manifest template.
+- Edit the configuration files. 
+
+Unless specified otherwise, the file you change is \appPackage\manifest.json.
+
+1. Copy the "$schema" and "manifestVersion" property values from the add-in's manifest to the corresponding properties of the Teams app's manifest template file.
+1. Adjust the "name.full", "description.short", and "description.full" property values as needed to take account of the fact that an Outlook add-in is part of the app. 
```

Comment:
```suggestion
1. Modify the `name.full`, `description.short`, and `description.full` property values as required to reflect that an Outlook add-in is part of the app. 
```

Reply:
This change is not consistent with the standard practice in Teams docs, which is to put manifest property names in quotation marks.